src/components/hero.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+const Hero = () => {
+  return (
+    <div class="pt-24">
+      <div class="container px-3 mx-auto flex flex-wrap flex-col md:flex-row items-center">
+        <div class="flex flex-col w-full md:w-2/5 justify-center items-start text-center md:text-left">
+          <p class="uppercase tracking-loose w-full">What business are you?</p>
```

Comment:
NAB: The following code block could use some formatting. It might be worth running your editors autoformat over this file until we include something like prettier in a later PR.

Reply:
belive it or not that's the cleanest it goes without prettier or something 🤔